src/discussion/views.py

```diff
@@ -113,6 +114,12 @@ def create(self, request, *args, **kwargs):
 
         response = super().create(request, *args, **kwargs)
         response = self.get_self_upvote_response(request, response, Thread)
+
+        created_thread = Thread.objects.get(id=response.data['id'])
```

Comment:
By default, `response` is a flat object without the nested attributed that we need on the frontend (in this case, `reviews`) Rather than making another call to get this data, I am using the serializer class to return properly serialized data (see return statement).

This way, all of our requests to the BE return a consistent object. ATM, creating a thread returns a different obj than fetching threads.

Reply:
I verified creating threads, comments, responses works as expected.